x-pack/platform/plugins/shared/streams/server/lib/streams/component_templates/logs_layer.ts

```diff
@@ -19,3 +29,186 @@ export const logsSettings: IndicesIndexSettings = {
     },
   },
 };
+
+export const baseFields: FieldDefinition = {
+  '@timestamp': {
+    type: 'date',
+  },
+  'stream.name': {
+    type: 'system',
+  },
+  'scope.dropped_attributes_count': {
+    type: 'long',
+  },
+  dropped_attributes_count: {
+    type: 'long',
+  },
+  'resource.dropped_attributes_count': {
+    type: 'long',
+  },
+  'resource.schema_url': {
+    type: 'keyword',
+  },
+  'scope.name': {
+    type: 'keyword',
+  },
+  'scope.schema_url': {
```

Comment:
Do we store scope attributes?

Reply:
Good point, that should be part of the base mapping, I'll add